test/e2e/specs/editor/various/autocomplete-and-mentions.spec.js

```diff
@@ -422,22 +422,44 @@ test.describe( 'Autocomplete (@firefox, @webkit)', () => {
 	test( 'should insert elements from multiple completers in a single block', async ( {
 		page,
 		editor,
-	} ) => {
-		await editor.canvas.click( 'role=button[name="Add default block"i]' );
-		await page.keyboard.type( '@fr' );
+	}, testInfo ) => {
+		const typingDelay = testInfo.project.name === 'webkit' ? 100 : 0;
```

Comment:
It might be a good idea to include a note about why we're adding the delay `webkit` and maybe even a link to this PR.

Reply:
Will add the note. I'll also rerun the test a couple of times in CI to ensure it's stable.

Reply:
Looks like it's not limited to webkit after all. While I couldn't make it fail locally for other browsers, CI does a great job at it! 😅   

I'll set the delay for all browsers then.

Reply:
Added missing comment in https://github.com/WordPress/gutenberg/pull/55093